json_schemas/manifest.json

```diff
@@ -23,9 +23,9 @@
             "pull_url": {"type": "string", "maxLength": 511, "format": "https-url"},
             "channelback_url": {"type": "string", "maxLength": 511, "format": "https-url"},
             "clickthrough_url": {"type": "string", "maxLength": 511, "format": "https-url"},
-            "healthcheck_url": {"type": "string", "maxLength": 511},
-            "about_url": {"type": "string", "maxLength": 511},
-            "dashboard_url": {"type": "string", "maxLength": 511},
+            "healthcheck_url": {"type": "string", "maxLength": 511, "format": "https-url"},
```

Comment:
htm, do we actually have a bug here, where `healthcheck_url` should be `health_check_url`? https://github.com/zendesk/zendesk_channels/blob/master/app/models/channels/any_channel/registered_integration_service.rb#L28

Reply:
@smedberg good pick up! Yep the column is called `health_check_url`, will update that.